.golangci.yaml

```diff
@@ -75,6 +75,8 @@ linters:
               desc: Use "go.opentelemetry.io/otel/semconv/v1.27.0" instead.
             - pkg: io/ioutil
               desc: Use the "io" and "os" packages instead. See https://go.dev/doc/go1.16#ioutil
+            - pkg: math/rand$
+              desc: Use the "math/rand/v2" package instead. See https://go.dev/doc/go1.22#math_rand_v2
```

Comment:
Any reason behind the switch to v2?

Reply:
No reason to stay _away_ from v1, really. The v2 `rand.N` is generic, so it can do `time.Duration` without casts; that was nice.